Libplanet.Store/MemoryStore.cs

```diff
@@ -134,8 +133,8 @@ BlockHash branchpoint
             }
         }
 
-        Transaction IStore.GetTransaction(TxId txid) =>
-            _txs.TryGetValue(txid, out object untyped) && untyped is Transaction tx
+        Transaction? IStore.GetTransaction(TxId txid) =>
+            _txs.TryGetValue(txid, out object? untyped) && untyped is Transaction tx
```

Comment:
Might as well change the type for `_txs` to `ConcurrentDictionary<TxId, Transaction>` or is there some other reason why this can't be done?

Reply:
It seems to be used only as a transaction type.